templates/simple_game/Makefile.macos.android

```diff
@@ -0,0 +1,302 @@
+#**************************************************************************************************
+#
+#   raylib makefile for Android project (APK building)
+#
+#   Copyright (c) 2017 Ramon Santamaria (@raysan5)
+#
+#   This software is provided "as-is", without any express or implied warranty. In no event
+#   will the authors be held liable for any damages arising from the use of this software.
+#
+#   Permission is granted to anyone to use this software for any purpose, including commercial
+#   applications, and to alter it and redistribute it freely, subject to the following restrictions:
+#
+#     1. The origin of this software must not be misrepresented; you must not claim that you
+#     wrote the original software. If you use this software in a product, an acknowledgment
+#     in the product documentation would be appreciated but is not required.
+#
+#     2. Altered source versions must be plainly marked as such, and must not be misrepresented
+#     as being the original software.
+#
+#     3. This notice may not be removed or altered from any source distribution.
+#
+#**************************************************************************************************
+
+# Define required raylib variables
+PLATFORM               ?= PLATFORM_ANDROID
+RAYLIB_PATH            ?= ../..
+
+# Define Android architecture (armeabi-v7a, arm64-v8a, x86, x86-64) and API version
+ANDROID_ARCH           ?= ARM
+ANDROID_API_VERSION     = 29
+ifeq ($(ANDROID_ARCH),ARM)
+    ANDROID_ARCH_NAME   = armeabi-v7a
+endif
+ifeq ($(ANDROID_ARCH),ARM64)
+    ANDROID_ARCH_NAME   = arm64-v8a
+endif
+
+# Required path variables
+# NOTE: JAVA_HOME must be set to JDK
+JAVA_HOME              ?= /Library/Java/JavaVirtualMachines/adoptopenjdk-13.0.1.jdk/Contents/Home
+ANDROID_HOME            = /Users/yunoinsky/Library/Android/sdk
```

Comment:
> It shouldn't contain user specific paths

@raysan5 Aaaaah! It's a stupid mistake, just too lazy to spend time on checking the pathway yesterday so i just use `which android`  to add the path variables here. Please forget it! 🤣 
I just replace it with `$(HOME)` variable so it can work on other macOS without edit the enviroment path

Reply:
nice! :D